### PR TITLE
[RFR] Added “Vanilla” entry

### PR DESCRIPTION
**Pull request by B1aZer**

### CODE REVIEW

**Comment by B1aZer** [comment]
Whoops! Apparently there is already a PR with Vanilla. Suppose you can disregard it 😊

File: README.md [hunk 1]
```diff
@@ -167,13 +167,13 @@ If you are willing to contribute, open a pull request to complete, update or fil
 ### V
 
 * [V8](/glossary/V8.md): Google’s open source javaScript engine. It’s what Chrome is running, but it’s also used for other projects like Node.js and MongoDB.
-* [Vanilla](/glossary/VANILLA.md): *to be completed.*
+* [Vanilla](/glossary/VANILLA.md): is a term for library/framework free JavaScript.
```
Comment (KittyGiraudel) [inline README.md]: Could we remove the _is_? :)

**Comment by B1aZer** [comment]
Removed 'is' in Vanilla entry.

**Comment by KittyGiraudel** [comment]
Thanks! :)

**Pull request merged**